scheduler/jobs/scheduleCommand.js

```diff
@@ -0,0 +1,66 @@
+const source = require('rfr');
+
+const logger = source('bot/utils/logger');
+const { sendMessage } = source('bot/utils/util');
+const Bot = source('bot');
+
+const { Settings } = source('models/settings');
+const { Statistics, EventTypes } = source('models/statistics');
+
+function handleRunCommand(job) {
```

Comment:
`handleScheduleCommand` instead?

Reply:
fixed in 3b66bcf41c99fa3390cf7d0c2bca01618b295549